### ISSUE TITLE
Add Dart support

### ISSUE THREAD

**Issue by joseluisq** [OctoLinker/OctoLinker#548]
It would be great to have Dart support too.

```dart
import 'package:test/test.dart'; // external
import 'package:local_package/lib.dart'; // local

void main() {
  // ...
}
```

**Comment by stefanbuck** [comment]
Yes! This would be super 👍 Thanks for the idea! Would you like to contribute to make it? The [ruby plugin](https://github.com/OctoLinker/OctoLinker/blob/master/packages/plugin-ruby/index.js) should provide you some hint on how to build it. I'm happy to assist you.

**Comment by joseluisq** [comment]
I would like to contribute as soon as I get free time again.
